### PR TITLE
NMS-15808: sync producer protobuf with horizon

### PR DESCRIPTION
**Pull request by RangerRick**
This PR syncs the protobuf file to match the latest Horizon, _plus_ pending changes related to `NMS-15808` to include new topology protocols.

### CODE REVIEW

**Comment by RangerRick** [comment]
@j-white @rssntn67 I don't know the details of the new `NODES`, `OSPFAREA`, and `NETWORKROUTER` protocols enough to know whether there's any specific work that needs to be done in ALEC to support them. Is it enough to just add the protocols to the list, or are there specific implications for correlation?

**Comment by j-white** [comment]
> @j-white @rssntn67 I don't know the details of the new `NODES`, `OSPFAREA`, and `NETWORKROUTER` protocols enough to know whether there's any specific work that needs to be done in ALEC to support them. Is it enough to just add the protocols to the list, or are there specific implications for correlation?

If we leave it as it, we still gain some value in that it will strengthen the relationship between the code nodes. Ideally, we would add some explicit handling though, and be deliberate about the graph it creates.

For reference, the logic sits in here: https://github.com/OpenNMS/alec/blob/develop/datasource/opennms-kafka/src/main/resources/inventory.groovy#L49

tl;dr; we're okay to do nothing else, if we do more we could get better correlation

**Comment by RangerRick** [comment]
@j-white great, thanks, I'll go ahead and make an issue to handle the new types and go ahead and merge this in the meantime

**Comment by RangerRick** [comment]
https://opennms.atlassian.net/browse/ALEC-269
